microsite/blog/2023-06-21-switching-out-sandbox.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Switching out the Software Templates Sandbox
+author: Ben Lambert
+authorURL: https://github.com/benjdlambert
+authorImageURL: https://avatars.githubusercontent.com/u/3645856?v=4
+---
+
+**TL;DR**: For the Backstage maintainers, ensuring that the project is highly secure for every adopter and end-user is one of our top priorities. With the recent vulnerabilities in the `vm2` sandbox, and have decided to move away from `vm2` and use `isolated-vm` instead to prioritize security for the Backstage end-users.
```

Comment:
```suggestion
**TL;DR**: For the Backstage maintainers, ensuring that the project is highly secure for every adopter and end-user is one of our top priorities. With the recent vulnerabilities in the `vm2` sandbox we have decided to move away from `vm2` and use `isolated-vm` instead, in order to ensure the security of the Backstage end-users.
```

Feeling "prioritize" is a bit weird here, it's not like we didn't do that before 😅